common/scaffolding/interactive-content-en.html

```diff
@@ -0,0 +1,387 @@
+---
+title: Embedded and interactive content - Scaffolding
+---
+---
+{
+	"title": "Embedded and interactive content - Scaffolding",
+	"language": "en",
+	"altLangPage": "interactive-Content-en.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-07-18"
+}
+---
+
+<h2>The <code>picture</code> element</h2>
+<picture></picture>
+<pre><code>&lt;picture&gt;&lt;/picture&gt;</code></pre>
+
+<h2>The <code>source</code> and <code>img</code> element</h2>
+<picture>
+	<source srcset="img/input-image-sample.jpg">
+	<img src="img/input-image-sample.jpg">
+</picture>
+<pre>
+<code>&lt;picture&gt;
+&lt;source srcset="img/input-image-sample.jpg"&gt;
+&lt;img src="img/input-image-sample.jpg"&gt;
+&lt;/picture&gt;</code>
+</pre>
+
+<h2>The <code>iframe</code> element</h2>
+<iframe src="https://www.canada.ca/en.html" title="The official website of the Government of Canada" width="700px" height="500px"></iframe>
+<pre><code>&lt;iframe src="https://www.canada.ca/en.html" title="The official website of the Government of Canada" width="700px" height="500px"&gt;&lt;/iframe&gt;</code></pre>
+
+<h2>The <code>embed</code> element</h2>
+<h3>An embedded image</h3>
```

Comment:
Let's add those 2 paragraph

Keep in mind that most modern browsers have deprecated and removed support for browser plug-ins, so relying upon `<embed>` is generally not wise if you want your site to be operable on the average user's browser.

Accessibility concerns: Use the title attribute on an embed element to label its content so that people navigating with assistive technology such as a screen reader can understand what it contains. The title's value should concisely describe the embedded content. Without a title, they may not be able to determine what its embedded content is. This context shift can be confusing and time-consuming, especially if the embed element contains interactive content like video or audio.

Reply:
Fixed in commit "Addressed changes".